rowcodec/encoder.go

```diff
@@ -299,10 +299,18 @@ var defaultStmtCtx = &stmtctx.StatementContext{
 }
 
 const (
-	rowKeyLen       = 19
-	recordPrefixIdx = 1
+	// Length of rowkey.
+	rowKeyLen = 19
+	// Index of record flag 'r' in rowkey used by master tidb-server.
```

Comment:
Could you explain why the length of recordPrefixIdx is 10?

Reply:
The rowkey format in master tidb is `t{8 bytes id}_r{8 bytes handle}`. The index of `r` is 10.